data-access/Models/UserModels.cs

```diff
@@ -7,8 +7,33 @@ public class User
     {
         public string UserName { get; set; }
         public uint UserId { get; set; }
+    }
+    
+    public class UserLoginResponse : User
+    {
+        public byte Activated { get; set; }
+        public string Email { get; set; }
+    }
+    
+    public class UserToken : User
+    {
         public string Token { get; set; }
     }
+
+    public class DetailedUserToken : UserToken
+    {
+        public string Forename { get; set; }
+        public string Surname { get; set; }
+        public string Organization { get; set; }
+        public string Email { get; set; }
+    }
+
+    public class UserEmail : User
+    {
+        public string Forename { get; set; }
+        public string Surname { get; set; }
+        public string Email { get; set; }
+    }
```

Comment:
I'm beginning to think there are too many user DTOs here, making it cumbersome. Let's not change it for now, but let's not add any new DTOs but rather organize these (or simplify them).